File: PelaWEB/TestesAutomatizados/Python/OddEven/test_number_operations.py
import number_operations

def test_is_even():
    assert number_operations.is_even(4) == True
    assert number_operations.is_even(5) == False
    assert number_operations.is_even(0) == True

def test_is_odd():
    assert number_operations.is_odd(3) == True
    assert number_operations.is_odd(-6) == False
    assert number_operations.is_odd(1) == True

File: PelaWEB/TestesAutomatizados/Java/src/test/java/Testes/test_number_operations.java

import static org.junit.Assert.assertTrue;

import java.Entradas.NumberOperations;

import static org.junit.Assert.assertFalse;
import org.junit.Test;

public class test_number_operations {
    @Test
    public void testIsEven() {
        NumberOperations numberOperations = new NumberOperations();
        assertTrue(numberOperations.isEvent(4));
        assertFalse(numberOperations.isEvent(7));
    }

    @Test
    public void testIsOdd() {
        NumberOperations numberOperations = new NumberOperations();
        assertTrue(numberOperations.isOdd(3));
        assertFalse(numberOperations.isOdd(6));
    }
}
